README.md

```diff
@@ -42,6 +42,19 @@ Fetch the API schema to confirm the server is up and running. (This doesn't requ
 
     curl http://localhost:8080/v3/api-docs.yaml
 
+### Running the server via `docker`
+
+If you wish to run the Terraware server and/or Postgres database via docker, check out the `./docker/Makefile`.
+
+#### Run both the server and database
+This is useful if you just want the server running locally for frontend development
```

Comment:
FYI, the frontend repo also has a docker-compose file for running the server locally, and that's the one the frontend developers typically use. Nothing wrong with also keeping the one in this repo up to date, though.